Python/Watermark_Images/README.md

```diff
@@ -0,0 +1,16 @@
+# Watermark Images
+
+## Description:
+Simple python scripts that can be used to add watermark to an image using either text or logo.
+
+## Requirements to run the notebook:
+
```

Comment:
Can we have a `requirements.txt` here.

And one more request, can you add the steps to run this as well, I understand that it's as basic as it gets, but for beginners, a proper step-wise layout is recommended.

Reply:
Used autopep8 to format the code. The blog was nice! Thank you for it.
Included steps to run the scripts and requirements.txt in the readme as suggested.